go.mod

```diff
@@ -8,7 +8,7 @@ require (
 	github.com/go-openapi/spec v0.19.3
 	github.com/go-openapi/validate v0.18.0 // indirect
 	github.com/gobuffalo/packr/v2 v2.7.1
-	github.com/google/go-cmp v0.3.1
+	github.com/google/go-cmp v0.3.1 // indirect
```

Comment:
How did you spot this? Or it was automatically included once you removed the direct dependencies?

Reply:
The dependency is resolved automatically by the go modules.